### ISSUE TITLE
NGINX realip module request

### ISSUE THREAD

**Issue by davidnewhall** [linuxserver/docker-letsencrypt#394]
Hello,

It looks like this container is missing the [nginx `realip` module](http://nginx.org/en/docs/http/ngx_http_realip_module.html). I put this behind cloudflare, and would like to use that module to log real IPs. Is there something I'm missing (as in, another way to log real IPs), or is adding that module something we can consider?

Thanks!

**Comment by gilbN** [comment]
Real ip does work. 
Look at the Cloudflare section in this post https://technicalramblings.com/blog/blocking-countries-with-geolite2-using-the-letsencrypt-docker-container/

**Comment by davidnewhall** [comment]
I swear I added those exact lines and got an error that realip was not a valid directive. Let me try again!

**Comment by davidnewhall** [comment]
haha, jeez. thanks, and sorry for the noise! Apparently adding semicolons to the end of the lines is a good idea.
